src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -279,7 +279,8 @@ private void AssertRevisionConfiguration(DatabaseRecord databaseRecord, LicenseS
             return;
 
         if (licenseStatus.CanSetupDefaultRevisionsConfiguration == false &&
-            databaseRecord.Revisions.Default != null)
+            databaseRecord.Revisions.Default != null &&
+            databaseRecord.Revisions.Default.Disabled == false)
```

Comment:
what is the implication of this? you are allowing to import disabled configuration, what can be done later on with it?

Reply:
I'm following other configurations like the expiration configuration:
https://github.com/ravendb/ravendb/blob/c1c7d5eb4ccd7841574ff3020335df5ea4a1a375/src/Raven.Server/ServerWide/ClusterStateMachine.License.cs#L316

You can import a disabled configuration, yes. You won't be able to turn enable it (this goes through the same method).